code/modules/shuttle/white_ship.dm

```diff
@@ -44,6 +44,12 @@
 /obj/machinery/computer/camera_advanced/shuttle_docker/whiteship/Initialize()

```

Comment:
it need in 
/obj/machinery/computer/camera_advanced/shuttle_docker/Initialize()